nautobot/core/graphql/generators.py

```diff
@@ -305,6 +306,11 @@ def generate_list_resolver(schema_type, resolver_name):
 
     def list_resolver(self, info, limit=None, offset=None, **kwargs):
         filterset_class = schema_type._meta.filterset_class
+
+        # Inverse of substitution logic from get_filtering_args_from_filterset() - transform "_description" back to "description"
+        if "_description" in kwargs:
+            kwargs["description"] = kwargs.pop("_description")
```

Comment:
2.x probably has a bug with `_type` filters due to the lack of an equivalent to this code.